libs/scully/src/lib/renderPlugins/launchedBrowser.ts

```diff
@@ -96,7 +96,14 @@ function obsBrowser(options: LaunchOptions = scullyConfig.puppeteerLaunchOptions
         filter(() => !isLaunching),
         // tap(() => log(green('relaunch cmd received'))),
         /** the long trottletime is to cater for the concurrently running browsers to crash and burn. */
-        throttleTime(15000)
+        delayWhen(() => {
+          let intervalLength = 15000;
+          if (!browser) {
+            intervalLength = 0;
+          }
+
+          return interval(intervalLength);
```

Comment:
Can you explain this to me?
Also, why an interval, doesn't this cause the browser the be relaunched every `intervalLength`?
I'm also unsure what `interval(0)` does. 
This code is in place to solve stability issues with puppeteer on systems with high-core-counts. Testing this part of the code is _really_ time-consuming, and I will not start on it unless I'm 100% convinced it has added benefits.

Reply:
Thank you for shedding some light around that part of the code, now it makes more sense. I left the throttling in place as it has to be there, refactored the code so that it's more optimal and added tests around my changes. I will push a new PR with the changes and more details. Please review and let me know if it makes more sense now.